README.rst

```diff
@@ -0,0 +1,132 @@
+Introduction
+============
+
+This package provides a contact contenttype.
+
+.. contents:: Table of Contents
+
+
+Installation
+------------
+
+Add the package as dependency to your setup.py:
+
+.. code:: python
+
+  setup(...
+        install_requires=[
+          'ftw.contacts',
+        ])
+
+or to your buildout configuration:
+
+.. code:: ini
+
+  [instance]
+  eggs += ftw.contacts
+
+and rerun buildout.
+
+Install the Generic Setup profile.
+
+Motivation
+----------
+
+The ``ftw.contacts`` modernize the very old egov.contactdirectory and
+prepares the package for plone 5 integration.
+
+Quick overview of the functions
+-------------------------------
+
+- Contactdirectory with ajax search functionality
+- Contact
+- KML-Representation of Contact
+- vCard-Representation of Contact
+- ftw.zipexport integration if you install the [zipexport]-extra
+- ftw.simplelayout integration if you install the [simplelayout]-extra
+
+How it looks
+------------
+
+The ``ftw.contacts`package provides a nice contactdirectory.
+
+.. image:: https://raw.github.com/4teamwork/ftw.contacts/master/docs/contactdirectory.png
+
+with a contact-view.
+
+.. image:: https://raw.github.com/4teamwork/ftw.contacts/master/docs/contact.png
+
+How it works
+------------
+
+Contactfolder
+~~~~~~~~~~~~~
+
+After installing the ``ftw.contacts`` package you can add a ``Contactfolder``to your plonesite.
+
+The ``Contactfolder``provides a nice overview of all avaiable contacts.
+Per default, the first 20 contacts will be shown and you can load more contacts by clicking on the
+``Load more``butten at the end of the list.
+
+You can search for contacts by filtering them trought the letters listed above the listing.
```

Comment:
I wouldn't call this "searching" :confused: maybe:
``The alphabetical filter allows the user to quickly find a contact. Furthermore contacts can be found by using the search field.`` :question: